### ISSUE TITLE
Message Signing Functionality - Does not exist

### ISSUE THREAD

**Issue by nthState** [ethereum/mist#3139]
- [√] I've asked for help in the [Mist Gitter](http://gitter.im/ethereum/mist) before filing this issue.

```
Version: `0.9.1`
OS & Version: macOS 10.12.6
Node version: `geth 0.0.0` 
Number of blocks synchronized: 0
```

Hi,

There is no message-signing functionality in the mist wallet client.

I'm expecting to see a UI component with an address picker, a text field where I can enter what I want signing, a button to sign the text and an output text field with the result.

1. Is there a reason for this?
2. Is there anything stopping me/someone adding this functionality and creating a pull-request for it?
3. I can see an old issue raised here asking for the functionality, but it hasn't had a response in a long while: [Issue 130](https://github.com/ethereum/mist/issues/130)

Kind Regards

Chris

**Comment by evertonfraga** [comment]
Currently, eth_sign prepends a string, for security reasons:
`sign(keccak256("\x19Ethereum Signed Message:\n" + len(message) + message)))`.

So, probably that wouldn't help much for your proposed use. 

After releasing 0.9.3 we'll look into the Signer, and we'll consider implementing this. 

Thank you for signaling.

**Comment by nthState** [comment]
Hi, Is this closed as it's going into 0.9.4?
